src/modules/personenkontext/domain/personenkontext-workflow.ts

```diff
@@ -127,14 +127,25 @@ export class PersonenkontextWorkflowAggregate {
         let allowedRollen: Rolle<true>[] = [];
         // If the user has rights for this specific organization or any of its children, return the filtered roles
         if (orgsWithRecht.includes(organisation.id)) {
-            const organisationMatchesRollenart: OrganisationMatchesRollenart = new OrganisationMatchesRollenart();
-            rollen.forEach(function (rolle: Rolle<true>) {
-                // Check here what kind of roles the admin can assign depending on the type of organisation
-                if (organisationMatchesRollenart.isSatisfiedBy(organisation, rolle) && !allowedRollen.includes(rolle)) {
-                    allowedRollen.push(rolle);
```

Comment:
This removes the check using OrganisationMatchesRollenart. Is that check already somewhere else?

Reply:
Yes it's part of the method checkReferences.

Reply:
Alright!